include/base/auto_ptr.h

```diff
@@ -35,24 +35,12 @@ template<typename T>
 using UniquePtr = std::unique_ptr<T>;
```

Comment:
I would vote to remove the `UniquePtr` alias now as well. Every once in a while we get copy/pasted code from ancient examples that incorrectly re-introduces `UniquePtr` usage, so it would be good to completely avoid that. I have a branch which does this, but it will likely conflict with this larger branch, so maybe better to do as part of this PR?

Reply:
jwpeterson/libmesh@9834ba282458c60bd96796d9f233d78f97c40916

Reply:
I agree; we've had it deprecated forever now.  I'd prefer a separate PR though.  I'll do that if I've inadvertently trampled on yours.